core/src/Select/Select.less

```diff
@@ -155,19 +144,6 @@
       background-color: @appBkgColor;
     }
 
-    // li.unselected {
-    //   color: #6b7994;
-    //   text-shadow: 1px 1px 0px #c9cdd6;
-    // }
-
-    // li strong {
```

Comment:
Removing this seems like it will negatively affect select options which contain HTML and the `<strong>` tag

Reply:
@shrunyan  this was more commented out code not in use, so  I cleaned it up